integration/consul_catalog_test.go

```diff
@@ -54,6 +54,18 @@ func (s *ConsulCatalogSuite) waitToElectConsulLeader() error {
 	})
 }
 
+func (s *ConsulCatalogSuite) waitForConnectCA() error {
+	return try.Do(15*time.Second, func() error {
```

Comment:
Not a big deal as it's in a test, but is there any way to reuse the RefreshInterval value from the Provider here?

Reply:
I do not know how to do that (short of adding a `const DefaultRefreshInterval` to the module)

Reply:
yeah don't worry about it.